### PR TITLE
main components of profile page finished implementing

### PR DESCRIPTION
**Pull request by yqj2k**

### CODE REVIEW

**Comment by yqj2k** [comment]
will merge into master at 9:45am if no reviewers.

File: src/App.tsx [hunk 1]
```diff
@@ -10,15 +9,16 @@ import Login from './pages/Login';
 import Signup from './pages/Signup';
 import Navbar from './components/Navbar';
 
+import ProfilePage from './pages/ProfilePage';
+
 function App() {
   return (
     <main>
-      <TempNavbar />
```
Comment (baconandchips) [inline src/App.tsx]: By not having the temp nav bar, I have found no easy way to navigate to yours or Sepand's pages

Comment (baconandchips) [reply src/App.tsx]: I would suggest keeping in the tempnavbar, at least for demonstration purposes in this sprint

File: src/pages/ProfilePage.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import ProfileMenu from './ProfileMenu';
+
+
+const useStyles = makeStyles((theme: Theme) => createStyles({
+
+    container: {
+        backgroundColor: "#2d46b9",
+    },
+    marginAutoContainer: {
+        width: 500,
```
Comment (baconandchips) [inline src/pages/ProfilePage.tsx:16]: I think the static containers/offsets make mobile (or smaller-ish, like inspector tab on the right side) sizing kinda wonky; but we can change later too if you'd like imo. Other than that, LGTM

File: src/pages/ProfilePage.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            <Grid container spacing={3} >
+                <Box className={classes.sideBoxes} />
+                <React.Fragment>
+                    <Box display="flex" width="50%" height="100vh" justifyContent="center" >
+                        <div className={classes.root}>
+                            <Grid container >
+                                <Grid item xs={12} justify="center" alignContent="center" alignItems="center" direction="column">
+                                    <Box marginLeft={50}>
+                                        <Box height="30vh">
+                                            <Avatar src="/broken-image.jpg" className={classes.largeImg}  />
```
Comment (baconandchips) [inline src/pages/ProfilePage.tsx:64]: I'm not 100% sure if this was your intent, but if you wanted to center-align the profile to the whitespace, you could try making the padding for a transparent box on top of the whitespace to have -200 or -100px, and just put the avatar into that space too, centered. Or just yeah make the box big enough to invlude the avatar too, and just stick it in there. Or if it doesn't matter then it's fine too

**Pull request merged**